articles/connectivity/conn-faq-psn-police.md

```diff
@@ -31,7 +31,7 @@ The PSN-P is based on the IPED (Inter-Provider Encryption Domain) to enable mult
 
 ## Can data get between the PSN for Policing CUG and the Enhanced Regime?
 
-Yes. There are gateways, external to UKCloud, that allow data to traverse the two networks. This is managed and controlled by Vodafone. For questions relating to this gateway please contact Vodafone or the PSN for Policing Authority.
+Yes. There are gateways, external to UKCloud which allow data to traverse the two networks. This is managed and controlled by Vodafone. For questions relating to this gateway please contact Vodafone or the PSN for Policing Authority.
```

Comment:
which allows

Reply:
"which allow" is correct, to agree with the plural "gateways"
However, I'm changing it back to "that" because the sentence doesn't make sense without the second second clause (https://www.grammarly.com/blog/which-vs-that/#:~:text=In%20a%20defining%20clause%2C%20use,and%20you%20can%20use%20which.)